packages/react-widgets/src/widgets/TimeSeriesWidget.js

```diff
@@ -172,14 +172,19 @@ function TimeSeriesWidget({
   const handleTimelineUpdate = useCallback(
     (timelinePosition) => {
       const { name: moment } = timeSeriesData[timelinePosition];
-      handleTimeWindowUpdate([
-        moment,
-        moment + STEP_SIZE_RANGE_MAPPING[selectedStepSize] * 1000
-      ]);
+      dispatch(
+        addFilter({
+          id: dataSource,
+          column,
+          type: FilterTypes.BETWEEN,
+          values: [moment, moment + STEP_SIZE_RANGE_MAPPING[selectedStepSize] * 1000],
```

Comment:
Why * 1000 here? is it milliseconds? It's not very evident... maybe you could add a const with a proper name here for the whole expression "const XXX = STEP_SIZE_RANGE_MAPPING[selectedStepSize] * 1000",